AbstractSQLOptimiser.ometajs

```diff
@@ -1,5 +1,22 @@
 var _ = require('lodash');
 
+var escapeForLike = function(str) {
+	return [
+		'Replace',
+		[	'Replace',
+			[	'Replace',
+			str,
```

Comment:
I think you want that indented one more level.

Reply:
Yeah you're right, I have the file recognised as JS because it's the best syntax highlighting I've found, but when moving code around the "invalid" ometa sections can confuse the auto indenter - fixed